framework/db/ColumnSchemaBuilder.php

```diff
@@ -46,17 +50,23 @@ class ColumnSchemaBuilder extends Object
      * @var mixed default value of the column.
      */
     protected $default;
+    /**
+     * @var Schema $schema Used schema
+     */
+    protected $schema;
```

Comment:
schema should not be coupled to this class. we already have DBMS specific implementations of ColumnSchemaBuilder.